DNN Platform/Website/Install/DotNetNuke.install.config.resources

```diff
@@ -4,7 +4,7 @@
     <script name="Schema">DotNetNuke.Schema</script>
     <script name="Data">DotNetNuke.Data</script>
   </scripts>
-  <version>08.00.04</version>
+  <version>09.04.04</version>
```

Comment:
This variable is connected to the initial database installation process. If it's updated both `DotNetNuke.Schema.SqlDataProvider` and `DotNetNuke.Data.SqlDataProvider` will need to be updated to match the specified version. My recommendation would be to revert this single line back to v8.0.4.

Reply:
I changed it back, sorry I was not intending on changing that version number I had changed it back to original but may have undone the change while making changes using ctrl + z.  Thank you for the catch.

Reply:
@ohine  when you going release that weather widget again??? I miss that module :)